ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutor.java

```diff
@@ -265,7 +267,23 @@ private void executeStatement(
       final CommandId commandId,
       final Optional<CommandStatusFuture> commandStatusFuture
   ) {
-    if (statement.getStatement() instanceof TerminateQuery) {
+    if (statement.getStatement() instanceof PauseQuery) {
+      pauseQuery((PreparedStatement<PauseQuery>) statement);
+
+      final String successMessage = "Query paused.";
```

Comment:
@vcrfxia Ah!  Here's what is returned when a query is run.  

Thoughts on this?  It follows what happens with `terminate`...

Reply:
Works for me! I'm all for slimmer code. Will take a look at your changes later today.

Reply:
Yeah... it is maybe wonky if one is using PAUSE ALL.  Also, if one repeatedly pauses or resumes a query, then there's no note that the query was already paused, etc.

Reply:
Hm good point. We could refactor `pauseQuery()` to return the success message, based on whether the query exists, whether it was "PAUSE ALL" etc? Could also do the same thing for terminate while we're at it.

I think that'd be a nice touch. Could do it in a follow-up instead of this PR if you prefer.